test/unittests/test_utils.py

```diff
@@ -12,6 +12,58 @@
 from crmsh import tmpfiles
 
 
+def test_check_ssh_passwd_need():
+    with mock.patch('crmsh.utils.get_stdout_stderr') as mock_get_stdout_stderr:
+        mock_get_stdout_stderr.side_effect = [(0, None, None), (1, None, None)]
+        assert utils.check_ssh_passwd_need(["node1", "node2"]) == True
+    mock_get_stdout_stderr.assert_has_calls([
+            mock.call('ssh -o StrictHostKeyChecking=no -o EscapeChar=none -o ConnectTimeout=15 -T -o Batchmode=yes node1 true'),
+            mock.call('ssh -o StrictHostKeyChecking=no -o EscapeChar=none -o ConnectTimeout=15 -T -o Batchmode=yes node2 true')
+        ])
+
+
+def test_get_member_iplist_None():
+    with mock.patch('crmsh.utils.get_stdout_stderr') as mock_get_stdout_stderr:
+        mock_get_stdout_stderr.return_value = (1, None, "Failed to initialize the cmap API. Error CS_ERR_LIBRARY")
+        assert utils.get_member_iplist() is None
+    mock_get_stdout_stderr.assert_called_once_with('corosync-cmapctl -b runtime.totem.pg.mrp.srp.members')
+
+
+def test_get_member_iplist():
```

Comment:
@liangxin1300 We should have a test case for the case when the command failst too.

Reply:
@diegoakechi we have this function to test the command failed: `def test_get_member_iplist_None`